src/check_datapackage/check.py

```diff
@@ -181,6 +182,11 @@ class for more details, especially about the default values.
     issues = exclude(issues, config.exclusions)
     issues = sorted(set(issues))
 
+    if os.getenv("CDP_DEBUG"):
```

Comment:
```suggestion
    # Use by doing `CDP_DEBUG=true uv run ...`
    if os.getenv("CDP_DEBUG"):
```

I don't think we need to externally document this.